settings.py

```diff
@@ -183,7 +183,28 @@
 # ex ADPOD_SLOTS = [1,2,3], will create 1st, 2nd and 3rd slot of adpod
 # ex ADPOD_SLOTS = [4,5], will create 4th and 5th slot of adpod
 # Slot numbers should be in incremental order.
-ADPOD_SLOTS = [1,2,3]
+ADPOD_SLOTS = [1]
+
+# ENABLE_DEAL_LINEITEM
+# THis option is only for Adpod setup
+# Set ENABLE_DEAL_LINEITEM = True for creating deal line item
+# Defaults to False
+ENABLE_DEAL_LINEITEM = False
+
+# DEAL_CONFIG - configuration for creating deal lineitem
+# THis option is only for Adpod setup
+# Provide price, dealtier prefix and mindealtier values
+# Number of LineItem created = len(prefix)*len(mindealtier) for each bidder
+# LineItem wil be created with pwtdt(dealtier)targeting with pwtdt value as prefix[index] + mindealtier[index] for each bidder
+# LineItem wil be created with pwtdt(dealtier)targeting with pwtdt value as prefix[index] + mindealtier[index] for each bidder
+# Set DFP_LINEITEM_TYPE = "SPONSORSHIP" and ENABLE_DEAL_LINEITEM = True for creating deal lineitem
+DEAL_CONFIG = {
+    "pubmatic":{
+        "price":10,
+        "prefix":["abc"],
+        "mindealtier":[5]
+    }
+}
```

Comment:
1. Is this default initialization really required? Can we move it comment section?
2. can we return error and expect user is providing the deal config?

Reply:
Default initialisation is not required.  If required values is not provided, error is returned.
I have moved this value to comment as sample